source/components/Autocomplete.js

```diff
@@ -139,15 +152,19 @@ export default class Autocomplete extends FormField {
   };
 
   updateSelectedWords = (event) => {
-    if (!this.props.maxSelections || (this.state.selectedWords.length < this.props.maxSelections && this.state.filteredWords && this.state.filteredWords.length > 0)) {
+    const canMoreWordsBeSelected = this.state.selectedWords.length < this.props.maxSelections;
+
+    if (!this.props.maxSelections || (canMoreWordsBeSelected && this.state.filteredWords && this.state.filteredWords.length > 0)) {
```

Comment:
You can shorten the second boolean too 😉 
`const areFilteredWordsAvailable = this.state.filteredWords && this.state.filteredWords.length > 0;`